Makefile

```diff
@@ -256,7 +256,7 @@ install_tools:
 	cd $(TOOLS_MOD_DIR) && go install golang.org/x/tools/cmd/goimports
 	$(warning Please run make docker_lint for lint. It is unreliable to use self-compiled golangci-lint. \
 		https://golangci-lint.run/usage/install/#install-from-source)
-	cd $(TOOLS_MOD_DIR) && go install github.com/golangci/golangci-lint/cmd/golangci-lint
+	cd $(TOOLS_MOD_DIR) && go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.0
```

Comment:
Are there any problems with using latest golangci?

Reply:
idk but 'make lint' didn't work for me. rolled back changes to this file